html-templates/cbl/student-dashboard.tpl

```diff
@@ -22,24 +23,42 @@
                 </tr>
             </thead>
 
-            <tbody></tbody>
+            <tbody>
+                <tr>
+                    <td class="scoring-domain-col">
+                        <span class="domain-skill">Cite Evidence</span>
+                        <div class="meta">
+                            <span class="domain-competency">Reading Literature, </span>
```

Comment:
This whole table is the "recent progress" widget which we had commented out for the initial version of the CBL tool we shipped to schools. Unlike the graduation progress bar, we do have the needed data to actually populate this, but I think @ckunkel should revisit its design as I'm not sure how it looks right now will mesh well with the data we actually have where each demonstration will have scores for a number of standards... should these be grouped by demonstration? should we show all standards for the most recent demonstration even if it makes this table really tall? should we show more than one demonstration worth of scores?